### DISCUSSION TITLE
Planned Release for Saleor Storefront 3.0 and Saleor SDK 3.0?

### DISCUSSION THREAD

**Discussion by TurkaljGoran** [saleor/saleor#7716] [General] [answered]
We've decided on using Saleor 3.0 (currently in beta) for the backend of our e-commerce project, but would also like to build the storefront using Saleor SDK 3.0 directly or with Saleor Storefront 3.0. 

Are you guys going to release new versions of Saleor Storefront and Saleor SDK together with Saleor 3.0 or in the weeks/months following the release of Saleor 3.0? 

Info on this would be very much appreciated! Cheers!

**Comment by karolkielecki** [comment, marked as answer]
@TurkaljGoran Hi there! There is no ETA of the final release of 3.0, but we recommend using the beta version of 3.0. Saleor Storefront will also be updated in tandem with Saleor Core--so don't worry about it. We have frozen the API, so there won't be any breaking changes.

**Reply by TurkaljGoran** [reply]
Thank you very much for the quick reply! :)
